src/Symfony/Bridge/Twig/Tests/Extension/EmojiExtensionTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Extension;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Extension\EmojiExtension;
+
+class EmojiExtensionTest extends TestCase
+{
+    /**
+     * @testWith ["😂", ":joy:"]
+     *           ["😂", ":joy:", "slack"]
+     *           ["😂", ":joy:", "github"]
```

Comment:
shouldn't those test with cases that use names that are only in one of those catalogs instead ? For now, tests would stay green even if we always use the slack catalogue in the implementation.

Reply:
Fixed